Adyen/Utilities/KeyboardObserver.swift

```diff
@@ -34,7 +37,8 @@ extension KeyboardObserver {
     }
 
     /// :nodoc:
-    public func removeObserver(_ observer: Any) {
-        NotificationCenter.default.removeObserver(observer)
+    public func stopObserving() {
+        keyboardObserver.map { NotificationCenter.default.removeObserver($0) }
```

Comment:
can be simplified as `keyboardObserver.map(NotificationCenter.default.removeObserver)`

Reply:
👍